README.rst

```diff
@@ -1,108 +1,42 @@
-Google Assistant Python Sample
-==============================
+Google Assistant SDK for Python
+===============================
 
-This repository contains a Python sample for the Google Assistant API.
+This package contains a collection of samples and tools to help you get started with the `Google Assistant SDK`_ using `Python`_.
```

Comment:
This file has a lot of content covered in more specific README.md files, and I fear will not scale when we have more examples.

I would remove the "Installing", "Authorization" and "Run the sample" sections, and instead have a section that explains the structure of the repo, links to the specific directories for the smaple and the bindings, and helps developers navigate.

Something like:

    Table of contents
    -----------------
    - [Google Assistant gRPC API bindings for Python](/google-assistant-grpc/): autogenerated bindings in Python to communicate with the Google Assistant gRPC API. 
    - [Push to talk sample](/googlesamples/assistant/grpc/): reference code for implementing a functional Assistant SDK client in python, including authorization, managing conversation state, and capturing and playing back audio.